### PR TITLE
Introduce a new config flag: sproxyd_endpoints

### PR DESCRIPTION
**Pull request by JordanP**
This will eventually replace sproxyd_host and sproxyd_path

### CODE REVIEW

**Comment by sbjorn** [comment]
This really makes sense.

Note that we have to update the scenario as well https://github.com/scality/ScalitySproxydSwift/blob/master/test/scenario/multi-backend/assets/saio/phase2/etc/swift/object-server/scality.conf#L16 to reflect this configuration change.

File: test/unit/test_server.py [hunk 1]
```diff
@@ -102,6 +102,18 @@ def check_api_compatible(name):
...
+    endpoint2 = 'http://h2:81/p'
+    conf = {'sproxyd_endpoints': ' , '.join([endpoint1, endpoint2])}
+    obj_serv = swift_scality_backend.server.app_factory(conf)
+
+    assert frozenset([
+        urlparse.urlparse(endpoint1),
+        urlparse.urlparse(endpoint2),
+    ]) == obj_serv._get_client_for_policy(0).read_clients[0]._endpoints
+
+
```
Comment (sbjorn) [inline test/unit/test_server.py]: I think it makes sense to add an additional test which asserts failure on attempt of configuration with malformed urls.

Comment (JordanP) [inline test/unit/test_server.py]: What do you mean by malformed URL ? Urlparse happily parses all sort of weird (malformed ?) urls

IMO validation should be done by our Sproxyd client (https://github.com/scality/scality-sproxyd-client) not by this driver. (so unit tests on this should be done in the sproxyd-client repo) What do you think ?

Comment (sbjorn) [inline test/unit/test_server.py]: Yes, I remember that now when you mention it. urlparse is really really stupid.

If this is just data that we pass as-is to the Sproxyd client, I agree that we don't need to do any validation here.

**Comment by sbjorn** [comment]
LGTM

**Pull request merged**